README.md

```diff
@@ -130,34 +130,31 @@ partial interface Navigator {
 };
 
 interface Ink {
-    Promise<InkPresenter> requestPresenter(DOMString type, optional Element presentationArea);
+    Promise<DelegatedInkTrailPresenter> requestPresenter(DOMString type, optional Element? presentationArea = null);
 }
 
 dictionary InkTrailStyle {
     DOMString color;
     unrestricted double diameter;
 }
 
-interface InkPresenter {
-}
-
-interface DelegatedInkTrailPresenter : InkPresenter {
+interface DelegatedInkTrailPresenter {
     void updateInkTrailStartPoint(PointerEvent evt, InkTrailStyle style);
     
-    readonly attribute Element presentationArea;
+    readonly attribute Element? presentationArea;
     readonly attribute unsigned long expectedImprovement;
 }
 ```
 
 ## Interface Details
 
-This proposal provides infrastructure for authors to request a generic `InkPresenter` interface. A delegated ink trail `InkPresenter` can be requested and provided by User Agents that support it. `requestPresenter` is intentionally made to be extensible via a parameter so that in the future, it can easily be extended to other types of ink presentation. For example, this could include fully delegated ink presentation, or a presenter than can handle complex tips via WebGL shaders.
+This proposal provides infrastructure for authors to request a `DelegatedInkTrailPresenter` interface from User Agents that support it. It is intentionally designed so that in the future, a generic `InkPresenter` interface could be introduced that `DelegatedInkTrailPresenter` would inherit from, providing greater extensibility. `requestPresenter` is made to be extensible via a parameter so that it can easily be extended to other types of ink presentation as well. For example, this could include fully delegated ink presentation, or a presenter than can handle complex tips via WebGL shaders.
 
-The `requestPresenter` method also takes in an optional `presentationArea` parameter, which is used by the User Agent to limit the visible area where the provided InkPresenter will take effect. This is necessary to prevent ink presentation outside of the provided area. If this argument is not provided, this will default to the containing viewport. This area is the provided element's bounding box, which does not require authors to recalculate if the element is moved or scrolled.
+The `requestPresenter` method also takes in an optional `presentationArea` parameter, which is used by the User Agent to limit the visible area where the provided InkPresenter will take effect. This is necessary to prevent ink presentation outside of the provided area. If this argument is not provided, this will default to the containing viewport. This area is the provided element's border-box, which does not require authors to recalculate if the element is moved or scrolled.
```

Comment:
for the 'border-box' piece, let's mention that the box is in client coordinates (typically, I don't believe boxes have the concept of origin, just dimensions)